src/test/ui/tool-attributes/tool-attributes-misplaced-1.stderr

```diff
@@ -3,24 +3,32 @@ error: cannot find derive macro `rustfmt` in this scope
    |
 LL | #[derive(rustfmt)]
    |          ^^^^^^^
+   |
+   = note: `rustfmt` is in scope, but it is a tool module
```

Comment:
Should we just leave out the notes for obscure DefKinds like tool modules? Which other DefKinds should be skipped?

Reply:
Ignoring tool modules now. All other Res/DefKinds seem fine (or impossible to reach through an import).